pdoc/doc.py

```diff
@@ -445,7 +445,7 @@ def _taken_from(self, member_name: str, obj: Any) -> tuple[str, str]:
 
         mod = _safe_getattr(obj, "__module__", None)
         qual = _safe_getattr(obj, "__qualname__", None)
-        if mod and qual and "<locals>" not in qual:
+        if mod and hasattr(qual, "__contains__") and "<locals>" not in qual:
```

Comment:
Do you think
```suggestion
        if mod and isinstance(qual, str) and "<locals>" not in qual:
```
would maybe be more appropriate here?

Reply:
Ah yes, nice catch 👍 (should have checked the signature)